gaphor/ui/diagrampage.py

```diff
@@ -40,6 +41,27 @@ def tooliter(toolbox_actions: Sequence[Tuple[str, Sequence[ToolDef]]]):
         yield from section
 
 
+with importlib.resources.path("gaphor.ui", "placement-icon-base.png") as f:
+    PLACEMENT_BASE = GdkPixbuf.Pixbuf.new_from_file_at_scale(str(f), 64, 64, True)
+
+_placement_pixbuf_map: Dict[str, GdkPixbuf.Pixbuf] = {}
+
+
+def get_placement_cursor(display, icon_name):
+    global _placement_pixbuf_map
```

Comment:
Why is the global required here, can we just define the `_placement_pixbuf_map` in the `get_placement_cursor` function?

Reply:
Indeed I think we can do without this. I’ll remove it.